src/service/poc_mobile.proto

```diff
@@ -119,6 +119,48 @@ message subscriber_location_ingest_report_v1 {
   subscriber_location_req_v1 report = 2;
 }
 
+message hotspot_threshold_report_req_v1 {
+  // The onchain address of the hotspot which has met the coverage
+  // threshold requirements
+  bytes hotspot_pubkey = 1;
+  // the threshold value representing the amount of transferred bytes the
+  // minimum number of subscribers as defined by subscriber_threshold have
+  // exceeded
+  uint64 bytes_threshold = 2;
+  // the threshold value representing the minimum number of subscribers
+  // that have exceeded the bytes_threshold for the hotspot
+  uint32 subscriber_threshold = 3;
+  // Unix timestamp in seconds when the thresholds were exceeded
+  uint64 threshold_timestamp = 4;
+  // pubkey of the carrier identity service
+  bytes carrier_pub_key = 5;
```

Comment:
this field is usually just called `pub_key` for consistency.

Reply:
I replicated this from the existing `subscriber_location_req_v1`